local-cli/runWindows/runWindows.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const chalk = require('chalk');
+const build = require('./utils/build');
+const deploy = require('./utils/deploy');
+
+/**
+ * Starts the app on a connected Windows emulator or mobile device.
+ * Options are the following:
+ *    root: String - The root of the application
+ *    debug: Boolean - Specifies debug build
+ *    release: Boolean - Specifies release build
+ *    arch: String - The build architecture (x86, x64, ARM, Any CPU)
+ *    desktop: Boolean - Deploy to the desktop
+ *    emulator: Boolean - Deploy to the emulator
+ *    device: Boolean - Deploy to a device
+ *    target: String - Device GUID to deploy to
+ */
```

Comment:
@mpodwysocki @rozele once we are done with the `rnpm cli` merge (ETA likely next RC), you should be able to define options like that: https://github.com/grabbou/react-native/blob/edae5458402b404925e595452a380eb778128811/local-cli/runAndroid/runAndroid.js#L176-L187